src/sage/libs/singular/option.pyx

```diff
@@ -28,7 +28,7 @@ this::
 
     sage: with opt_ctx(red_tail=False, red_sb=False):
     ....:    std(I)[-1]
-    d^2*e^6 + 8*c^3 + ...
+    d^2*e^6 +...8*c^3 + ...
```

Comment:
One of these is supposed to be tail-reduced and the other not... we could do e.g. `reduced = std(I)[-1]` in the first case and `notreduced = std(I)[-1]` in the second, with the test then being `reduced != notreduced`?

Reply:
OK. In this case it was still being tested since the last element of the reduced basis doesn't contain `8*c^3` but this makes it a bit more future proof.